start.R

```diff
@@ -22,7 +22,9 @@ helpText <- "
 #'
 #'    --help, -h:        show this help text and exit
 #'    --debug, -d:       start a debug run with cm_nash_mode = debug
-#'    --gamscompile, -g: compile gms of all selected runs
+#'    --gamscompile, -g: compile gms of all selected runs. Combined with
+#'                       --interactive, it given the user time to fix errors;
```

Comment:
> it given the user time to fix errors

Say what?

Reply:
is that clearer now?

Combined --interactive, it stops in case of compilation errors, allowing the user to fix them and rerun gamscompile;

Reply:
Yes.  And also it's a sentence.

Reply:
when I write about compilation errors, my brain automatically switches to introducing syntax errors, I guess.